try:
    raise Exception('delete this file')
except Exception as e:
    print("HI", e)
    pass
